wallet-new/stack.yaml

```diff
@@ -0,0 +1,24 @@
+resolver: lts-9.1
+
+packages:
+- '.'
+
+# Unreleased support for servant-0.12
+- location:
+    git: https://github.com/haskell-servant/servant-quickcheck.git
+    commit: 6e6595f68c947107beba185392b350abd2009ba0
+  extra-dep: true
+
+extra-deps:
+  - universum-0.7.0
+  - serokell-util-0.5.0
+  - acid-state-0.14.3
+  - log-warper-1.3.3
+  - time-units-1.0.0
+  - fmt-0.5.0.0
+  - servant-0.12
+  - servant-server-0.12
+  - servant-client-0.12
+
+nix:
+  shell-file: ../shell.nix
```

Comment:
Hm, I think it's better to have only one `stack.yaml` per repository.

Reply:
Right, I agree. This is a relic from the prototype.